examples/config-gc-periodic.json

```diff
@@ -2,15 +2,23 @@
     "distSpecVersion": "1.1.0-dev",
     "storage": {
         "rootDirectory": "/tmp/zot",
-        "gc": true,
-        "gcDelay": "1h",
-        "gcInterval": "24h",
+        "gc": {
```

Comment:
Don't put this under "gc"
^ This is mainly understood for cleanup after user-triggered DELETEs

Create a new section called "retention" 
^ This is mainly understood for internally-triggered DELETEs.